src/plugins/intel_gpu/src/graph/graph_optimizer/concat_input_order.cpp

```diff
@@ -140,12 +140,20 @@ void concat_input_order::run(program& p) {
         bool single_format = true;
         std::vector<tensor::value_type> feature_sizes;
         feature_sizes.reserve(inputs_count);
+        bool continue_outer = false;
```

Comment:
shouldnt we add some other condition in line 122 that input is dynamic shape?

Reply:
Done.